### ISSUE TITLE
How is this compared to mcfgthread?

### ISSUE THREAD

**Issue by 44iCdnz** [meganz/mingw-std-threads#90]
https://github.com/lhmouse/mcfgthread/

**Comment by nmcclatchey** [comment]
That will depend on the aspects that you are comparing.

- Performance should be fairly similar for all mutually-supported versions of Windows, though it will vary depending on number of threads, amount of contention, and the version of Windows for which you are compiling. (Note: I have not benchmarked this)
- Compatibility is a bit different; mingw-std-thread supports versions of Windows that mcfgthread does not. In particular, mingw-std-threads supports Windows XP and Vista, while mcfgthread does not. 
- Ease of use varies; mingw-std-thread can be used simply by including the headers, while (as I recall) mcfgthread requires modifying the compiler. mingw-std-thread also does not require calling a specialized exit function (see mcfgthread's notes section in the readme).

In short, if you need XP or Vista compatibility, choose mingw-std-thread. If not, use whatever you think will minimize bugs (eg. forget to call `__MCF_exit()`) or maintenance when writing library code (eg. conditionally including the `mingw.*.h` headers based on whether Windows is the target).
